src/app/_components/Header/index.js

```diff
@@ -1 +1 @@
-export { default } from './Header';
+export { Header } from './Header';
```

Comment:
🟠 aren't we using default imports for components? can we stick to named/default everywhere? ( i prefer named )
🟢 can `Header` folder be converted to just `Header.js` file with component?

Reply:
![image](https://github.com/keenethics/mayak/assets/57231356/1694097c-bad8-4b06-bc2e-56560272aee2)